src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +212,30 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('button');
```

Comment:
We can create an `enum` with the valid options (close, back), verify it's valid, and then use that same attribute below without having to check again which button we are referring to. 

(example of an enum)
https://github.com/ampproject/amphtml/blob/eaaa44e21a281ebf853a3637f885f258f391af52/src/amp-story-player/amp-story-player-impl.js#L43-L48

So for example (pseudocode, but you get the idea):

```
   const buttonConfig = this.element_.getAttribute('button');
   if (!BUTTONS_ENUM.contains(buttonConfig) { return; }
   ...
   createCustomEvent(this.win_, BUTTONS_ENUM[buttonConfig], {})
   
 

```